### DISCUSSION TITLE
General setting "Show current working directory changes as an artificial commit." seems to be removed.

### DISCUSSION THREAD

**Discussion by aomorishouko** [gitextensions/gitextensions#9262] [Q&A] [answered]
I can't find a settings point to disable showing changes as an artificial commit. In version 3.4 it was in the General settings. (https://git-extensions-documentation.readthedocs.io/en/release-3.4/settings.html). Inside the current documentation (https://git-extensions-documentation.readthedocs.io/en/release-3.5/settings.html) it's still listed but the setting doesn't seem to be there:
![Screenshot (6)](https://user-images.githubusercontent.com/85352018/120808638-8cc5f480-c549-11eb-927e-6c216db89a29.png)
My exact version is Git Extensions 3.5.0.11713 Build 0a5ef9ca6ce87754a510aa02c7a96d8b915714ac (Dirty).

Has the setting been moved somewhere else?

**Comment by gerhardol** [comment, marked as answer]
![image](https://user-images.githubusercontent.com/6248932/120811013-da436100-c54b-11eb-938c-edaee6650e21.png)
